### ISSUE TITLE
Clients not playing

### ISSUE THREAD

**Issue by AfroPsycho** [badaix/snapcast#823]
Hi I have a setup with rpi 4B (Server) and 2 pi zero w (clients)
I did install rapotify to server and use librespot to route sound to the clients
When I reboot clients (BedRoom & WorkRoom) BedRoom starts playing but it stops when WorkRoom starts after a while ()generally less than 2 minutes) work room stops playing too

Pi Zeros are connected to bluetooth speakers with pi-btaudio it is normally working great
Also I know that music is still playing because I can see it in the web interface and can play in web browser but no sound from zeros

I know I should share the logs but I don't know where to find them I am bit noob at this things

**Comment by Mistico83** [comment]
I have a similar issue with Volumio (Version 2.861, Tue Dec 22 14:39:52 CET 2020).

If the first stream I play on Volumio, the local snapclient and mobile clients works as usual. It's a Pi Model B, connected through HDMI to a A/V Receiver via snapclient at localhost (Pi acts as a server+client), and clients with snapclient.

If I stop one stream, and start another, I must reconnect clients. Also the localhost client, wich misses connection and no output is passed to HDMI.

How can I help debugging this? Here's the output of snapclient on desktop PC when I stop playback:

```
2021-03-09 12-58-06 [Info] Chunk: -2    -2      -2      -2      17      118
2021-03-09 12-58-07 [Info] Chunk: -2    -2      -2      -2      51      115
2021-03-09 12-58-08 [Info] Chunk: -3    -2      -2      -2      84      109
2021-03-09 12-58-09 [Info] Chunk: 0     0       -2      -2      117     110
2021-03-09 12-58-10 [Info] Chunk: 3     3       0       -1      150     111
2021-03-09 12-58-11 [Info] Chunk: 3     4       3       0       184     112
2021-03-09 12-58-11 [Info] Failed to get chunk
2021-03-09 12-58-16 [Notice] No chunk received for 5000ms. Closing ALSA.
```

The snapclient on PC do resume playback, neither the Pi or the Android clients. Everything is up to date. Any idea?

**Comment by badaix** [comment]
@AfroPsycho in the bug report template that you've deleted is written not only that you should attach debug logs and how you get them, but also that you should report the version, beyond other things. I don't even know what rapotify is. 
Hint: it seems that the Rpi0 has problems using WiFi and BT in parallel as discussed in another issue. And please consider changing your username, something without Psycho suffix, I'm really having a hard time to reply to Psychos.
@Mistico83 last time when I checked the repository of this official volumio pluigin, it shipped a Snapclient v0.15. Please come back when you can reproduce the issue with a plain Snapcast setup. I'm not related in any way with volumio or this plugin.
